requirements.txt

```diff
@@ -1,9 +1,10 @@
 dateparser==1.0.*
-pre-commit==2.9.*
+pre-commit==2.10.*
 pytest==6.2.*
-pytest-cov==2.10.*
-pytest-mock==3.4.*
+pytest-cov==2.11.*
+pytest-mock==3.5.*
 python-dateutil==2.8.*
 pytz==2019.*
```

Comment:
Shall we bump `pytz` as well?

Reply:
Sadly we cannot until dateutil is updated with the new tz database. Else, our parsing tests that go through all possible timezones fails (pytz 2020.x ships a newer db than dateutil).